front/src/applications/stdcm/components/StdcmResults/StdcmDebugResults.tsx

```diff
@@ -53,32 +53,36 @@ const StdcmDebugResults = ({
 
   return (
     <>
-      {projectedData && pathProperties.manchetteOperationalPoints && (
-        <ResizableSection
-          height={manchetteWithSpaceTimeChartHeight}
-          setHeight={setManchetteWithSpaceTimeChartHeight}
-          minHeight={MANCHETTE_WITH_SPACE_TIME_CHART_DEFAULT_HEIGHT}
-        >
-          <div
-            className="osrd-simulation-container mb-2"
-            style={{
-              height: manchetteWithSpaceTimeChartHeight,
-            }}
+      {projectedData &&
+        projectedData?.spaceTimeData.length > 0 &&
```

Comment:
Follow-up: osrd-ui shouldn't crash when `spaceTimeData` is empty. Could we open an issue to track this bug?

Reply:
Here is the [bug](https://github.com/OpenRailAssociation/osrd/issues/10496)

Reply:
Thanks!